src/Dapper.Oracle/Dapper.Oracle.csproj

```diff
@@ -3,7 +3,7 @@
     <Import Project="..\common.targets" />
     <PropertyGroup>
         <IsTestProject>false</IsTestProject>
-        <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
```

Comment:
See other comment, add net461 as explicit target

Reply:
Added net461 to the target frameworks and committed.